engine/api/project.go

```diff
@@ -40,6 +43,17 @@ func (api *API) getProjectsHandler() Handler {
 			if err != nil {
 				return sdk.WrapError(err, "getProjectsHandler")
 			}
+
+			if strings.ToUpper(withPermissions) == "W" {
+				res := make([]sdk.Project, 0, len(projects))
+				for _, p := range projects {
+					if p.Permission >= 5 {
```

Comment:
this constant https://github.com/ovh/cds/blob/master/engine/api/permission/permission.go#L11